### PR TITLE
Update post installation document for fluent-package

### PR DESCRIPTION
**Pull request by kenhys**
This PR is follow-up of #462

### CODE REVIEW

**Comment by kenhys** [comment]
Fixed a typo.

File: installation/post-installation-guide.md [hunk 1]
```diff
@@ -1,9 +1,39 @@
...
+
+After editing the configuration file, restart `fluentd` using `systemctl` command:
+
+```text
+$ sudo systemctl restart fluentd
+```
+
+#### Logging
+
+By default, `td-agent` writes its operation logs to the following file:
```
Comment (daipom) [inline installation/post-installation-guide.md]:
```suggestion
By default, Fluentd writes its operation logs to the following file:
```

File: installation/post-installation-guide.md [hunk 2]
```diff
@@ -90,6 +120,14 @@ The following configuration uses the `in_forward` plugin as an input source and
 
 Fluentd manages plugins as Ruby gems but stores them in their dedicated directory separated from the normal Ruby gems.
 
+#### `fluent-package`
+
+A special program `fluent-gem` is used to manage plugin gems. For example, the following command installs a plugin to connect to S3 \(including both `in_s3` and `out_s3` plugins\):
+
+```text
+ $ sudo /usr/sbin/fluent-gem install fluent-plugin-s3
```
Comment (daipom) [inline installation/post-installation-guide.md]:
```suggestion
 $ sudo fluent-gem install fluent-plugin-s3
```

Would a full path be better?
I know the existing other descriptions use the full path. 
If unifying them would be better, that's fine.

**Review by daipom — APPROVED** [review]
Thanks! Basically, looks good to me.
I have commented on some points.

**Comment by kenhys** [comment]
Fixed DCO issue.

**Pull request merged**